.circleci/config.yml

```diff
@@ -57,9 +57,11 @@ commands:
                 command: |
                   Test/scripts/ci-integration.sh
             - store_test_results:
-                path: ./integration-tests/test-results
+                path: /home/circleci/project/integration-tests/test-results
             - store_artifacts:
-                path: ./integration-tests/screenshots
+                path:  /home/circleci/project/integration-tests/screenshots          
+            - store_artifacts:
+                path: /home/circleci/project/integration-tests/test-results
```

Comment:
Is this additional `store_artifacts` line an error?

Reply:
it stores screenshots and wdio logs